webapi/src/test/scala/org/knora/webapi/http/ServerVersionE2ESpec.scala

```diff
@@ -43,8 +43,15 @@ class ServerVersionE2ESpec extends E2ESpec(ServerVersionE2ESpec.config) {
         "return the custom 'Server' header with every response" in {
             val request = Get(baseApiUrl + s"/admin/projects")
             val response: HttpResponse = singleAwaitingRequest(request)
-            // logger.debug(s"response: ${response.toString}")
+            println(s"response: ${response.toString}")
+            logger.debug(s"response: ${response.toString}")
```

Comment:
Please remove the println and logger.debug. 
Hint: You can easily use the InteliJ debugger for checking the values.

Reply:
> Please remove the println and logger.debug.
> Hint: You can easily use the InteliJ debugger for checking the values.

done and breakpoints are working :-)